### PR TITLE
[web-ui] #357 fixes

### PR DESCRIPTION
**Pull request by uidev547**
fix for #357 issue

### CODE REVIEW

File: _includes/search-welcome-component.html [hunk 1]
```diff
@@ -3,16 +3,16 @@
 		<div class="welcome-component">
 			<div class="welcome-comp-header">
 				<h1>KodeBeagle</h1>
-				<p>Contextual code search* by <a href="http://www.imaginea.com" target="_blank">Imaginea</a></p>
+				<p>Contextual code search* over xxxxxxxx lines of code by <a href="http://www.imaginea.com" target="_blank">Imaginea</a></p>
```
Comment (hashd) [inline _includes/search-welcome-component.html]: @ScrapCodes Can you give the exact number for this to @uidev547.

@uidev547 Make the change then and add to PR

File: about/index.html [hunk 2]
```diff
@@ -13,9 +13,9 @@
 				<div class="welcome-component">
 					<div class="welcome-comp-header">
 						<h1>KodeBeagle</h1>
-						<p>Contextual <a href="{{ page.basepath }}">code search</a> by <a href="http://www.imaginea.com" target="_blank">Imaginea</a></p>
+						<p>Contextual <a href="{{ page.basepath }}">code search*</a> over xxxxxxxx lines of code by <a href="http://www.imaginea.com" target="_blank">Imaginea</a></p>
```
Comment (hashd) [inline about/index.html]: Same as above

File: _includes/search-welcome-component.html (hunk 1 above)
Comment (ScrapCodes) [inline _includes/search-welcome-component.html]: @amit-shar Has them.

Comment (amit-shar) [inline _includes/search-welcome-component.html]: @uidev547 lines of code  920670735

Comment (hashd) [inline _includes/search-welcome-component.html]: Wow, thats quite a number. Lets ceil it to 1 Billion lines of code (Yeah, I know I'm bad at ceiling :))

File: _includes/search-result/statistics.html [hunk 3]
```diff
@@ -31,9 +31,11 @@
                 </span>
 
                 <div style="margin-left: 20px; display: inline-block" class="pull-right">
+                    <span class="inline-block">code snippets</span>
```
Comment (hashd) [inline _includes/search-result/statistics.html]: `Code Snippet`

File: _includes/search-result/statistics.html [hunk 4]
```diff
@@ -31,9 +31,11 @@
...
                     <div class="btn-group" role="group">
-                      <button type="button" ng-click="toggleCodeSnippets($event)" class="btn btn-default" ng-class="{'active' : model.toggelSnippet}">show code snippets</button>
-                      <button type="button" ng-click="toggleCodeSnippets($event)" class="btn btn-default" ng-class="{'active' : !model.toggelSnippet}">hide code snippets</button>
+                      
+                      <button type="button" ng-click="toggleCodeSnippets($event)" class="btn btn-default" ng-class="{'active' : model.toggelSnippet}">show</button>
```
Comment (hashd) [inline _includes/search-result/statistics.html]: `Show`

File: _includes/search-result/statistics.html [hunk 5]
```diff
@@ -31,9 +31,11 @@
...
+                      <button type="button" ng-click="toggleCodeSnippets($event)" class="btn btn-default" ng-class="{'active' : !model.toggelSnippet}">hide</button>
```
Comment (hashd) [inline _includes/search-result/statistics.html]: `Hide`

**Pull request merged**